app/pages/Help.js

```diff
@@ -19,56 +20,361 @@ class Help extends React.Component {
+                    Mumu for Windows{' '}
+                  </a>
+                  , make sure to download the 64 bit verision under the colored button.
+                </li>
+                <li>
+                  <a href="https://adl.netease.com/d/g/a11/c/mac" target="_blank">
+                    Mumu for Mac OS{' '}
+                  </a>
+                  (Not yet compatible with M1).
+                </li>
+                <li>
+                  <a href="https://www.bignox.com/en/download/fullPackage/win_64?beta" target="_blank">
+                    Nox app player (64-bit) Windows.{' '}
+                  </a>
+                </li>
+                <li>
+                  <a href="https://www.bignox.com/en/download/fullPackage/mac_fullzip?beta" target="_blank">
+                    Nox app player (64-bit) MacOS.{' '}
+                  </a>
+                </li>
+                <li>
+                  <a href="https://drive.google.com/file/d/1mQuivBo2lpRvPI4Obapvcb9NYDcaSUoh/view" target="_blank">
+                    Mumu setup guide{' '}
+                  </a>
+                </li>
+              </ul>
+            </p>
+          </Accordion.Content>
+
+          {/* mumu setup */}
+          <Accordion.Title active={activeIndex === 2} index={2} onClick={this.handleAccordionClick.bind(this)}>
+            <Icon name="dropdown" />
+            Mumu Setup
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 2}>
+            <p>
+              <a href="https://drive.google.com/file/d/1mQuivBo2lpRvPI4Obapvcb9NYDcaSUoh/view" target="_blank">
+                Mumu setup guide
+                {/* install  cert > start proxy > set proxy settings in mumu > start game */}
+              </a>
+              <ol>
+                <li> SWEX get cert </li>
+                <li> Open mumu shared folder</li>
+                <li> Copy cert from swex in shared folder.</li>
+                <li> Navigate to: Settings → Security → Install from SD card → $MuMuSharedFolder</li>
+                <li>
+                  Select the generated cert (ca.pem), name it whatever you like and set Credential use to VPN & apps. Tap okay.
+                  <ul>
+                    <li>If the cert file is greyed out for some reason try to rename the extension from .pem to .cer</li>
+                  </ul>
+                </li>
+                <li> Start the proxy in SWEX</li>
+                <li> Navigate to: Settings → WiFi</li>
+                <li> Long press listed network → modify network</li>
+                <li> Change "Proxy" to "Manual"</li>
+                <li> server: ip listed in SWEX, usually 192.168.x.x; Port: Port listed in SWEX, default = 8080</li>
+                <li> Press Save</li>
+                <li> In SWEX: confirm HTTPS is turned on</li>
+                <li> In SWEX: Start proxy</li>
+              </ol>
+            </p>
+          </Accordion.Content>
+
+          {/* Nox setup */}
```

Comment:
Not a huge fan of these meaningless comments, since you can see the section directly under it anyway. Would suggest to remove these.